Terminal.Gui/Views/ScrollBarView.cs

```diff
@@ -40,7 +40,7 @@ public ScrollBarView ()
         WantContinuousButtonPressed = true;
         ClearOnVisibleFalse = false;
 
-        Added += (s, e) => CreateBottomRightCorner (e.Parent);
```

Comment:
I missed these before.

Please use named methods instead of anonymous methods for event subscriptions.

Unsubscription cannot be guaranteed with anonymous.

The `(s,e) => ` is what's creating that anonymous method.

Reply:
`ScrollBarView.cs` will be removed later, so isn't a problem for now.